### PR TITLE
Issue #118

### PR DESCRIPTION
**Pull request by kruchas91**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,6 +19,24 @@
...
+   *
+   * We do this here as browsers have "interesting" ways of parsing code in `eval` in js files, which causes the entire file to fail parsing instead 
+   * of just the offending string.
+   */ 
+
+  try {
+    eval("x => 0");
+  } catch (e) {
+    if (e instanceof SyntaxError) {
+      alert("This application is known to work with Firefox 35 and above.");
```
Comment (Yoric) [inline index.html:35]: I only asked you to remove the second sentence. Please restore the first one :)

**Comment by jordiC91** [comment]
I must be really tired cause now i don't understand what you mean.

**Comment by kruchas91** [comment]
Please explain where you would like to see the sentence. I don't understand.

**Comment by Yoric** [comment]
Let's try:

``` js
alert("This application requires a browser that implements ES6, such as a recent version of Firefox. Your browser is apparently missing some ES6 features.");
```

**Pull request closed without merging**